packages/buckets/src/buckets.spec.ts

```diff
@@ -60,7 +61,7 @@ describe('Buckets...', () => {
     expect(res.root).to.not.be.undefined
     expect(res.item?.isdir).to.be.true
     // @todo: Should we rename itemsList to just be items?
-    expect(res.item?.itemsList).to.have.length(0)
+    expect(res.item?.itemsList).to.have.length(1) // Includes .textileseed
```

Comment:
These returns now include the .textileseed value, so we have to bump these all up by one